clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2010,10 +2064,6 @@ class SyclKernelPointerHandler : public SyclKernelFieldHandler {
     return true;
   }
 
-  // FIXME: Array of pointers/ array of types containing pointers
-  // will be handled in a follow-up PR. Currently they continue to
-  // trigger decomposition.
-
 public:
   QualType getNewType() {
```

Comment:
Maybe it is worth to rename it to `getNewRecordType()` so anyone who tries to get array type won't into a problem with the fact that there is a special function for arrays.